src/__tests__/testid-test.tsx

```diff
@@ -288,3 +289,12 @@ test('Meter test ids', () => {
         },
     ]);
 });
+
+test('Drawer test ids', () => {
+    checkTestIds(<Drawer title="Title" subtitle="Subtitle" description="Description" onClose={() => {}} />, [
+        {
+            componentName: 'Drawer',
+            internalTestIds: ['title', 'subtitle', 'description'],
```

Comment:
are these defined in the spec?

Reply:
Not in spec, but I was assuming the same testids for these elements that we were using in cards, etc